lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -1429,6 +1430,56 @@ int parse_mdmev(const char *at_response, enum lte_lc_modem_evt *modem_evt)
 	return -ENODATA;
 }
 
+int parse_rai(const char *at_response, struct lte_lc_rai_cfg *rai_cfg)
+{
+	struct at_parser parser;
+	int err;
+	int tmp_int;
+
+	err = at_parser_init(&parser, at_response);
+	__ASSERT_NO_MSG(err == 0);
+
+	/* Cell Id */
+	err = string_param_to_int(&parser, AT_RAI_CELL_ID_INDEX, &tmp_int, 16);
+	if (err) {
+		LOG_ERR("Could not parse cell_id, error: %d", err);
+		goto clean_exit;
+	}
+
+	if (tmp_int > LTE_LC_CELL_EUTRAN_ID_MAX) {
```

Comment:
Is it expected that modem sends the notification with an invalid cell ID? If this is not expected behavior, do we want to send the event with an invalid cell ID, or should we rather just ignore the notification?

Reply:
I hope they don't send it with invalid CELL ID but I don't know. I can ignore it.

Reply:
Ignored.